org.eclipse.jdt.core/model/org/eclipse/jdt/internal/core/JavaCorePreferenceInitializer.java

```diff
@@ -63,6 +63,7 @@ public void initializeDefaultPreferences() {
 		defaultOptionsMap.put(JavaCore.COMPILER_TASK_CASE_SENSITIVE, JavaCore.ENABLED);
 		defaultOptionsMap.put(JavaCore.COMPILER_DOC_COMMENT_SUPPORT, JavaCore.ENABLED);
 		defaultOptionsMap.put(JavaCore.COMPILER_PB_FORBIDDEN_REFERENCE, JavaCore.ERROR);
+		defaultOptionsMap.put(JavaCore.COMPILER_IGNORE_UNNAMED_MODULE_FOR_SPLIT_PACKAGE, JavaCore.DISABLED);
```

Comment:
On second look, I don't understand why this is added (under the heading "Override some compiler defaults"). Default is already disabled, or am I missing smth?

Reply:
Ah, it's not necessary.